src/save_arrays.c

```diff
@@ -267,7 +267,7 @@ struct UnkStruct_0202E474 *sub_020270C4(SaveData *saveData) {
     return SaveArray_Get(saveData, SAVE_UNK_23);
 }
 
-PC_STORAGE *GetStoragePCPointer(SaveData *saveData) {
+PC_STORAGE *Save_PCStorage_Get(SaveData *saveData) {
```

Comment:
Few things:
1. Why `Save_PCStorage_Get` and not `Save_GetPCStorage`?
2. Shouldn't this be `SaveArray_PCStorage_Get`/`SaveArray_GetPCStorage`
3. `SaveArray_Get` is probably a bad name in itself, it's more like `SaveArray_GetEntry` (`SaveArray_Get` implies getting the entire SaveArray itself)

Reply:
1. consistent with all the other getters
2. probably
3. idk